helix-core/src/main/java/org/apache/helix/controller/rebalancer/waged/WagedRebalancer.java

```diff
@@ -444,6 +517,9 @@ private void refreshBaseline(ResourceControllerDataProvider clusterData,
         optimalAssignment.getOptimalResourceAssignment();
 
     LOG.info("Finish calculating. Time spent: {}ms.", System.currentTimeMillis() - startTime);
```

Comment:
Nit: I usually find it more helpful to just say:

"Finish calculateAssignment(). Took: {} ms.". Just to make it consistent with other logs.

Reply:
This message was not written in this PR. So I don't think this PR needs to change it.

Reply:
Will you create another PR to change it then? I think it would be easier to modify it in the PR since it's in the scope.

Reply:
Yes. A new PR is submitted for resolving this.